<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет вектора Стокса</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExperiments"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCalculate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Расчет</string>
    </property>
    <addaction name="actionParameters"/>
    <addaction name="actionRefraction"/>
    <addaction name="separator"/>
    <addaction name="actionLoadData"/>
    <addaction name="separator"/>
    <addaction name="actionCalculate12"/>
    <addaction name="actionCalculate3"/>
    <addaction name="separator"/>
    <addaction name="actionPlot"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Логи</string>
    </property>
    <addaction name="actionCalculationLog"/>
    <addaction name="separator"/>
    <addaction name="actionLogDispRays"/>
    <addaction name="actionLogNatRays"/>
    <addaction name="actionLogPolarization"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalculate"/>
   <addaction name="menuLog"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Отключиться</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Параметры</string>
   </property>
  </action>
  <action name="actionIntervals">
   <property name="text">
    <string>Интервалы</string>
   </property>
  </action>
  <action name="actionLoadData">
   <property name="text">
    <string>Загрузить данные</string>
   </property>
  </action>
  <action name="actionLoadData3">
   <property name="text">
    <string>Загрузить данные (задача 3)</string>
   </property>
  </action>
  <action name="actionCalculate12">
   <property name="text">
    <string>Расчет параметров излучения</string>
   </property>
  </action>
  <action name="actionCalculate3">
   <property name="text">
    <string>Определение типа покрытия</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>График</string>
   </property>
  </action>
  <action name="actionCalculationLog">
   <property name="text">
    <string>Протокол расчета</string>
   </property>
  </action>
  <action name="actionLogDispRays">
   <property name="text">
    <string>Лог расчета рассеянного излучения</string>
   </property>
  </action>
  <action name="actionLogNatRays">
   <property name="text">
    <string>Лог расчета естественного излучения</string>
   </property>
  </action>
  <action name="actionLogPolarization">
   <property name="text">
    <string>Лог расчета поляризации</string>
   </property>
  </action>
  <action name="actionGradient">
   <property name="text">
    <string>Тест метода градиента</string>
   </property>
  </action>
  <action name="actionNewExperiment">
   <property name="text">
    <string>Новые эээээксперименты</string>
   </property>
  </action>
  <action name="actionOpenExperiment">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionOpenRefraction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImportRefraction">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExportRefraction">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionRefraction">
   <property name="text">
    <string>Показатели преломления</string>
   </property>
  </action>
  <action name="actionExportRefraction_2">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImportRefraction_2">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExperiments">
   <property name="text">
    <string>Эксперименты</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
